docs/guide/migrating-to-version-9.md

```diff
@@ -570,6 +570,18 @@ Removed as of `v9`. Use `KBreadcumbs` instead.
 
 ### KTruncate
 
+#### Props
+
+* `isTextContent` prop has been changed to `textContent`
```

Comment:
the naming of this prop is a bit confusing; I would think I would pass in a string to `textContent` - is there a better name we could use?